src/fides/api/models/property.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from sqlalchemy import Column, String
+from sqlalchemy.ext.declarative import declared_attr
+
+from fides.api.db.base_class import Base
+from fides.api.db.util import EnumColumn
+from fides.api.schemas.property import PropertyType
+from fides.config import get_config
+
+CONFIG = get_config()
+
+
+class Property(Base):
+    @declared_attr
+    def __tablename__(self) -> str:
+        return "plus_property"
+
```

Comment:
nit: given my [comment](https://github.com/ethyca/fidesplus/pull/1326#discussion_r1503232707) on the plus PR re: duplication error handling, you _could_ override the `create` method of the base class and handle "name" and "key" duplications in a way here that makes it clearer to report on them in that endpoint function.

not a must-have, just thinking about how to make that error handling as simple as possible while providing the user with sensible informtation!

Reply:
I think for now I'll just separate `KeyOrNameAlreadyExists` exceptions from other exceptions and keep this fancier check for a future iteration